scene/resources/mesh.cpp

```diff
@@ -848,27 +827,19 @@ void _fix_array_compatibility(const Vector<uint8_t> &p_src, uint32_t p_old_forma
 					if ((p_old_format & OLD_ARRAY_COMPRESS_NORMAL) && (p_old_format & OLD_ARRAY_FORMAT_TANGENT) && (p_old_format & OLD_ARRAY_COMPRESS_TANGENT)) {
```

Comment:
In 3.x we had the option to use octahedral encoding with 16 bit values and 8 bit values for additional memory bandwidth savings - is this something we will want to support here as well (either now or in the future?)

Reply:
Maybe at some point, but in general this adds a lot of complexity and not a lot of win because on the devices where you save the most bandwidth on mobile you rarely use normalmapping anyway, and this requires different shader versions to decode that, which are far more expensive on Vulkan (pipelines..) so for now probably better not.